Makefile

```diff
@@ -72,22 +72,26 @@ TMP_FILES := bpf/kmesh/bpf2go/bpf2go.go \
 	bpf/kmesh/ads/include/config.h \
 	bpf/include/bpf_helper_defs_ext.h \
 
-.PHONY: all
-all:
+.PHONY: all data daemon
+all: data daemon
+
+data:
 	$(QUIET) find $(ROOT_DIR)/mk -name "*.pc" | xargs sed -i "s#^prefix=.*#prefix=${ROOT_DIR}#g"
 
 	$(QUIET) make -C api/v2-c
 	$(QUIET) make -C bpf/deserialization_to_bpf_map
 	
 	$(QUIET) $(GO) generate bpf/kmesh/bpf2go/bpf2go.go
-	
+
+	$(call printlog, BUILD, "kernel")
+	$(QUIET) make -C kernel/ko_src
+
+daemon:
```

Comment:
This is not only daemon, it also includes other I would still call this `all`.

Reply:
what about call this controller?